policyengine_uk/variables/household/wealth/total_wealth.py

```diff
@@ -8,4 +8,11 @@ class total_wealth(Variable):
     value_type = float
     unit = GBP
 
-    adds = ["property_wealth", "corporate_wealth"]
+    adds = [
```

Comment:
are you sure these are MECE? @nikhilwoodruff

Reply:
Lets keep savings in but drop the ones after